connection/src/traits.rs

```diff
@@ -116,6 +116,20 @@ impl From<LastBlockInfoResponse> for BlockInfo {
     }
 }
 
+impl From<BlockInfo> for LastBlockInfoResponse {
+    fn from(src: BlockInfo) -> Self {
+        let mut result = LastBlockInfoResponse::new();
+        result.index = src.block_index;
+        result.network_block_version = src.network_block_version;
+        result.set_minimum_fees(HashMap::from_iter(
+            src.minimum_fees
+                .iter()
+                .map(|(token_id, fee)| (**token_id, *fee)),
```

Comment:
Might as well use `into_iter` to consume the object

```suggestion
                .into_iter()
                .map(|(token_id, fee)| (*token_id, fee)),
```

Reply:
thanks, will apply these fixes